### PR TITLE
👀 New accessibility module

### PR DESCRIPTION
**Pull request by hbernardi-luna**
## 📜 Description
Add a new module for Accessibility:
- `Modifier` extensions for accessibility
- `AccessibilityCheckBoxRow` and the stateless version of this `Composable`
- `AccessibilityState` to handle `Talkback` activation
- `openAccessibilitySettings` to open directly accessibility settings page

Also, CI has been updated to deploy automatically all modules if needed (i.e if version has changed). Once merging on develop, lbcaccessibility should be available on Maven Snapshot Repository.

## 💡 Motivation and Context
Facilitate accessibility in our developments.

## 💚 How did you test it?
With the demo app.

## 📝 Checklist
* [x] I compiled before submitting the PR
* [x] I reviewed the submitted code
* [x] I launched `./gradlew detekt`

## 📸 Screenshots / GIFs
https://user-images.githubusercontent.com/72007948/167801121-4e496e85-0dfe-4440-9707-7cbfe6af95c8.mov

### CODE REVIEW

File: app/src/main/java/studio/lunabee/compose/accessibility/AccessibilityScreen.kt [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+                FloatingActionButton(onClick = openAccessibilitySettings) {
+                    Icon(
+                        painter = painterResource(id = R.drawable.ic_accessibility),
+                        tint = MaterialTheme.colors.primary,
+                        contentDescription = null, // decorative, button will be moved to TopAppBar if talkback is activate.
+                    )
+                }
+            }
+        }
+    ) {
```
Comment (PublisherLunabee) [inline app/src/main/java/studio/lunabee/compose/accessibility/AccessibilityScreen.kt:101]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/LunabeeStudio/Lunabee_Compose_Android/blob/0da7e67017640b337b066a3410a30c89cff519b0/app/src/main/java/studio/lunabee/compose/accessibility/AccessibilityScreen.kt#L101"></span>Content padding parameter it is not used</td>
    </tr>
  </tbody>
</table>

**Comment by hbernardi-luna** [comment]
@rlatapy-luna I just added a commit to put dependencies in the generated pom.xml, it will avoid implementing dependencies by the client (tested on Prismea), and I update Compose and Kotlin, beta is like stable for Compose 🤓)

File: lbcaccessibility/proguard-rules.pro [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+#   public *;
+#}
+
+# Uncomment this to preserve the line number information for
+# debugging stack traces.
+#-keepattributes SourceFile,LineNumberTable
+
+# If you keep the line number information, uncomment this to
+# hide the original source file name.
+#-renamesourcefileattribute SourceFile
```
Comment (rlatapy-luna) [inline lbcaccessibility/proguard-rules.pro:21]: Can be remove

File: lbcaccessibility/src/main/java/studio/lunabee/compose/lbcaccessibility/composable/AccessibilityCheckboxRow.kt [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+ * - Checkbox is not clickable
+ * - Row handle checked state with [androidx.compose.foundation.selection.toggleable]
+ *
+ * This Composable is stateless, checked value has to be handled.
+ *
+ * @param currentCheckedStateValue current value for [androidx.compose.material.Checkbox]
+ *
+ * @param onCheckedChange on user interaction
+ *
+ * @param accessibilityDescription see [AccessibilityDescription]
```
Comment (rlatapy-luna) [inline lbcaccessibility/src/main/java/studio/lunabee/compose/lbcaccessibility/composable/AccessibilityCheckboxRow.kt:94]: Lot of blank line with blank line between `@Param` 😛

File: lbcaccessibility/src/main/java/studio/lunabee/compose/lbcaccessibility/extension/ContextExt.kt [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import android.content.Intent
+import android.provider.Settings
+
+/**
+ * Open phone settings to accessibility page.
+ */
+fun Context.openAccessibilitySettings() {
+    val intent = Intent(Settings.ACTION_ACCESSIBILITY_SETTINGS)
+    startActivity(intent)
+}
```
Comment (rlatapy-luna) [inline lbcaccessibility/src/main/java/studio/lunabee/compose/lbcaccessibility/extension/ContextExt.kt:34]: Not really linked to compose, and I'm not we need an extension for that (I mean, we could create an extension for every `Settigns.ACTION_XXX`...)

File: lbcaccessibility/src/main/java/studio/lunabee/compose/lbcaccessibility/composable/AccessibilityCheckboxRow.kt [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+ * @param horizontalArrangement default [Arrangement.Horizontal] for the [Row]
+ *
+ * @param verticalAlignment default [Alignment.Vertical] for the [Row]
+ *
+ * @param rowContent custom content to display with the [Checkbox] in a [RowScope]
+ */
+@Composable
+fun AccessibilityCheckBoxRow(
+    initialCheckStateValue: Boolean,
+    accessibilityDescription: AccessibilityDescription,
```
Comment (rlatapy-luna) [inline lbcaccessibility/src/main/java/studio/lunabee/compose/lbcaccessibility/composable/AccessibilityCheckboxRow.kt:62]: `initialCheckStateValue` & `AccessibilityDescription.StateDescription.currentStateValue` looks like the same data set twice.
I'm not sure StateDescription should embed the current value

File: versions.properties [hunk 6]
```diff
@@ -1,5 +1,5 @@
 #
-# Copyright � 2022 Lunabee Studio
+# Copyright � 2022 Lunabee Studio
```
Comment (rlatapy-luna) [inline versions.properties:2]: again 🤔

**Review by rlatapy-luna — CHANGES_REQUESTED** [review]

File: versions.properties (hunk 6 above)
Comment (hbernardi-luna) [reply versions.properties:2]: I think it is changed by refreshVersion each time 😬

**Review by rlatapy-luna — APPROVED** [review]

**Pull request merged**